Signal/src/ViewControllers/ConversationView/ConversationCollectionView.m

```diff
@@ -12,6 +12,11 @@ @implementation ConversationCollectionView
 
 - (void)setFrame:(CGRect)frame
 {
+    if (frame.size.width == 0 || frame.size.height == 0) {
+        // Ignore iOS Auto Layout's tendency to temporarily zero out the
+        // frame of this view during the layout process.
+        return;
+    }
```

Comment:
Is this a perf optimization?

I'm a little worried about skipping the `super` call on such fundamental layout logic - but it'll be exercised enough that any terrible bug should be exposed pretty quickly.

Reply:
I've added a comment that explains.  This (I believe perfectly safe) hack spares us from all sorts of worse hacks which used to use - which never worked well anyway.